pkg/products/monitoring/reconciler.go

```diff
@@ -180,6 +194,13 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
+	phase, err = r.ReconcileNamespace(ctx, r.Config.GetFederationNamespace(), installation, serverClient)
```

Comment:
@pb82 Leaving a comment for what was discussed on a call.
This ReconcileNamespace fn will set a label of `monitoring-key:middleware` on the namespace, which causes any prometheus operator CRDs in the namespace to be reconciled by the rhmi prometheus operator instance.
This manifests as 1 new ServiceMonitor being reconciled by both the rhmi & cluster prometheus operators.

Recommendation is to have specific logic in this file for creating the namespace so that generic rhmi stuff doesn't get applied to it.